codegen/src/main/scala/akka/grpc/gen/Main.scala

```diff
@@ -35,11 +35,16 @@ object Main extends App {
 
   private val generateServer: Boolean = !reqLowerCase.contains("generate_server=false")
 
-  private val generatePlay: Boolean = reqLowerCase.contains("generate_play=true")
-
-  private val serverPowerApis: Boolean = reqLowerCase.contains("server_power_apis=true")
-
-  private val usePlayActions: Boolean = reqLowerCase.contains("use_play_actions=true")
+  /**
+   * For example `akka.grpc.gen.scaladsl.play.PlayScalaClientCodeGenerator`
+   */
+  val ExtraGeneratorsRegex = """(?:.*,)extra_generators=([^,]+)(?:,.*)?""".r
+  private val extraGenerators: List[String] = req.getParameter match {
+    case ExtraGeneratorsRegex(generators) =>
+      generators.split(";").toList
+    case _ =>
+      List.empty
+  }
```

Comment:
Reaching a point where it would make sense to parse req into a `Map` perhaps

Reply:
agree but will do in a different PR